apps/accounts/validators.py

```diff
@@ -85,20 +85,29 @@ class PasswordReuseAndMinAgeValidator(object):
 
     def __init__(self,
                  password_min_age=60 * 60 * 24,
-                 password_reuse_interval=60 * 60 * 24 * 120):
+                 password_reuse_interval=60 * 60 * 24 * 120,
+                 password_expire=0):
 
         msg1 = "Invalid OPTIONS, password_min_age < password_reuse_interval expected, " \
                "but having password_min_age({}) >= password_reuse_interval({})"
+        msg2 = "Invalid OPTIONS, password_expire < password_reuse_interval expected, " \
+               "but having password_expire({}) >= password_reuse_interval({})"
+        msg3 = "Invalid OPTIONS, password_min_age < password_expire expected, " \
+               "but having password_expire({}) >= password_reuse_interval({})"
```

Comment:
(nit, optional) I think this is easier to read if we have these messages nested under the if statements below. It's a small enough function that it works either way, just don't see any readability or functional benefit to having these listed out before the logic of the conditionals below.